config/environments/development.rb

```diff
@@ -36,7 +36,7 @@
     port: 587,
     user_name: ENV["GMAIL_USERNAME"],
     password: ENV["GMAIL_PASSWORD"],
-    authentication: :plain,
+    authentication: :login,
```

Comment:
giải thích cho a sao lại thay đổi thế?

Reply:
Tại hôm trước e test mail thì nó báo account not accepted nên e search google tìm cách sửa thì nó bảo sửa thế này ạ. nhưng lỗi không phải ở đây với lại e thấy cũng không khác gì nên không sửa lại. có cần để lại như cũ k ạ?

Reply:
thế để lại như cũ đi :smile: